org.ekstep.questionunit.mcq-1.2/renderer/template/mcq-layouts.js

```diff
@@ -445,17 +459,21 @@ MCQController.grid2.postRender = function (question) {
     if (question.data.options.length < 4) {
         MCQController.grid2.adjustOptions(question);
     }
+    var defaultMaxHeight = $(".mcq2-2-option").outerHeight();
+    var defaultMaxWidth = $(".mcq2-2-option").width();
+    var size = MCQController.getMaxWidthHeight(question.data.options, defaultMaxHeight, defaultMaxWidth);
+    $(".border-solid").css("height", size.maxHeight);
```

Comment:
$(".border-solid").css("height", size.maxHeight);
    $(".border-solid").css("width", size.maxWidth);

3 times repeated the same, Why dont you move this to common function(getMaxWidthHeight) rename the function name updateImageWH